app/preparation.py

```diff
@@ -0,0 +1,14 @@
+def get_ready(knights_config: dict) -> dict:
+    for person in knights_config.keys():
+        king = knights_config[person]
```

Comment:
Why king?

Reply:
u should change king to knight everywhere, cause in future u should write accurate names of variables